modules/consul-cluster/README.md

```diff
@@ -13,25 +13,34 @@ This folder defines a [Terraform module](https://www.terraform.io/docs/modules/u
 code by adding a `module` configuration and setting its `source` parameter to URL of this folder:
 
 ```hcl
+#demo consul cluster
 module "consul_cluster" {
-  # TODO: update this to the final URL
-  # Use version v0.0.5 of the consul-cluster module
+  # Use version v0.0.5 of the consul-cluster module (or check latest tags)
   source = "github.com/hashicorp/terraform-aws-consul//modules/consul-cluster?ref=v0.0.5"
 
   # Specify the ID of the Consul AMI. You should build this using the scripts in the install-consul module.
-  ami_id = "ami-abcd1234"
-  
+  ami_id = "ami-6820c012" # Ubuntu AMi, do not use in producuction
+  instance_type = "t2.medium"
+  cluster_name = "${var.cluster_name}"
+  cluster_size = "${var.num_servers}"
+  ssh_key_name = "${aws_key_pair.auth.key_name}"
```

Comment:
We actually intentionally try to keep the number of parameters small (see the comment below about checking `vars.tf`) or this example code quickly goes out of date as parameters are added/removed/etc. For the full example, it's always best to look at the working example code in the actual `.tf` files.